staging/src/k8s.io/apiserver/pkg/endpoints/installer.go

```diff
@@ -506,6 +506,8 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 		Subresource: subresource,
 		Kind:        fqKindToRegister,
 
+		DecodeToGroupVersion: schema.GroupVersion{Group: fqKindToRegister.Group, Version: runtime.APIVersionInternal},
```

Comment:
This is the hub version, right?

Reply:
yeah

Reply:
was trying to avoid naming this "InternalVersion" to avoid confusion. ideas welcome. InMemoryHandlerVersion?

Reply:
> yeah

request a rename to `HubGroupVersion`.  Also, `GroupVersion` or `GroupVersionKind`?

Reply:
renamed. everywhere this is used, only the GroupVersion is used, so kind information would be discarded.

Reply:
I got used to HubGroupVersion.